src/containers/RapidAppointment/RapidAppointment.tsx

```diff
@@ -287,22 +287,25 @@ export function RapidAppointment() {
             },
             {
               name: "Reasoning",
-              value: reasoningString,
+              value: reasoningString === "" ? "Not Reported" : reasoningString,
```

Comment:
Any chance this could be any other falsy value at runtime? I'm on my phone at the moment, but generally I would write ternary's condition like `!reasoningString ? ...`

Reply:
I don't believe so, as it is defined as let `reasoningString = "";` and the only operations that are happening are strings getting appended.